### ISSUE TITLE
multiple evalNWBin always execute just the first *.bin

### ISSUE THREAD

**Issue by Autumn2016** [nwjs/nw.js#5215]
this is my source code:
[https://github.com/Autumn2016/NWDemo](https://github.com/Autumn2016/NWDemo)

when i do like this:
```
    //first evalNWBin
	var one = require('nw.gui').Window.get().evalNWBin(null, 'one.bin')();
	console.log(one);
	console.log(one.sum());
	
     //second evalNWBin
	var two = require('nw.gui').Window.get().evalNWBin(null, 'two.bin')();
	console.log(two);
	console.log(two.moon());
```
question:
the first evalNWBin is work well,but the second evalNWBin is not well.
the result of the second evalNWBin is the same with the first.

![image](https://cloud.githubusercontent.com/assets/18275891/17695163/ec9c9d7c-63d9-11e6-8b17-e725e22e1398.png)

if you can help me, i will be deeply gratefu!l

**Comment by richardg867** [comment]
Can confirm. v0.16.1 on Linux x86

**Comment by ghostoy** [comment]
merged to #5168
